pkg/k8s-client/client.go

```diff
@@ -12,7 +12,9 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"k8s.io/client-go/tools/clientcmd"
+
+	dconfig "github.com/Mellanox/ib-kubernetes/pkg/config"
```

Comment:
IIRC, usually external packages are renamed in case of collision. is there a need to rename here ?
(or was this renamed before removing L15 ?)